### ISSUE TITLE
q-tooltip component is actually using wrong font-size style.

### ISSUE THREAD

**Issue by e274426380** [quasarframework/quasar#17137]
### What happened?

.q-tooltip--style says font-size:10px, but the example style in the documentation actually has a font-size of 12px.
In the actual application, the font-size is only 10px.
10px is very, very small, which makes the actual style completely different from the documented style.

### What did you expect to happen?

I think I need to change the font-size of the .q-tooltip--style to 12px to match the actual use case in the documentation.

### Reproduction URL

https://codepen.io/rstoenescu/pen/xxEvBLR

### How to reproduce?

```
<div id="q-app">
    <span>
    here
    <q-tooltip>
     tooltips
    </q-tooltip>
  </span>
</div>
```

On this page you can also find the css style for font-size:10px, while the actual application style is 12px, which is contradictory

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

**Comment by myleslee** [comment]
Agree. The font size is currently too small which makes people squint.

**Comment by tysongf** [comment]
You can use quasar text classes:

```html
<q-tooltip class="text-body1">Regular Font</q-tooltip>

<q-tooltip class="text-h6">Large Font</q-tooltip>
```

Custom classes affecting font size need to be applied to the default slot of the q-tooltip component:

```html
<q-tooltip>
    <template v-slot:default>
        <span class="super-big-font"> Lock Vault </span>
    </template>
</q-tooltip>
```

**Comment by rstoenescu** [comment]
Thank you for the report, but after verification this is intended behavior, not a bug.

The claimed contradiction does not exist: the CSS served by the docs site is exactly `.q-tooltip--style { font-size: 10px }`, with no override anywhere, so docs tooltips render at the same 10px as any default Quasar app. The only docs example with a bigger tooltip font is the Coloring example, which explicitly applies the `text-body2` class (14px). The reproduction codepen also shows the default tooltip behaving exactly as documented.

The 10px default is deliberate. It follows the Material Design 2 tooltip spec (10sp on desktop, 14sp on mobile), which Quasar matches with `$tooltip-fontsize: 10px` plus `$tooltip-mobile-fontsize: 14px` on small screens.

If you prefer a larger size (Material Design 3 moved plain tooltips to 12sp, for example), it is a one-line customization in your app: set the `$tooltip-fontsize` Sass variable, or add a class to the tooltip as shown above in this thread. Changing the framework default would visibly alter every existing app, so we are keeping it as is.
